daemons/dss-notify-v2/Readme.md

```diff
@@ -30,9 +30,12 @@ Bundle creation notifications have the format
 
 ```
 {
+  'dss-api': dss_api
```

Comment:
Please change this to `'dss_api': {dss_api_url}`.

Note the underscore, which is consistent with the rest of the event schema.